src/query/functions/tests/it/scalars/datetime.rs

```diff
@@ -35,6 +35,42 @@ fn test_datetime() {
     test_timestamp_arith(file);
     test_to_number(file);
     test_rounder_functions(file);
+    test_convert_timezone(file);
+}
+
+fn test_convert_timezone(file: &mut impl Write) {
```

Comment:
Cool.

SELECT CONVERT_TIMEZONE(
  'America/Los_Angeles',
  'America/New_York',
  to_timestamp('2024-01-01 14:00:00')
) AS conv;

SELECT CONVERT_TIMEZONE(
  'Europe/Warsaw',
  'UTC',
  '2024-01-01 00:00:00'::TIMESTAMP
) AS conv;

SELECT CONVERT_TIMEZONE(
  'America/Los_Angeles',
  '2024-04-05 12:00:00 +02:00'::TIMESTAMP
) AS time_in_la;

SELECT
  CURRENT_TIMESTAMP() AS now_in_la,
  CONVERT_TIMEZONE('America/New_York', CURRENT_TIMESTAMP()) AS now_in_nyc,
  CONVERT_TIMEZONE('Europe/Paris', CURRENT_TIMESTAMP()) AS now_in_paris,
  CONVERT_TIMEZONE('Asia/Tokyo', CURRENT_TIMESTAMP()) AS now_in_tokyo;

Need to add these tests in logical test.